data/adapters/ontologies_adapter.py

```diff
@@ -106,21 +103,9 @@ def process_file(self):
             self.outputs[t]['secondary'].close()
 
     def process_ontology(self):
-        print('Downloading {}...'.format(self.ontology))
-
-        with tempfile.NamedTemporaryFile(delete=False) as temp_file:
-            with urllib.request.urlopen(Ontology.ONTOLOGIES[self.ontology]) as response:
-                with gzip.GzipFile(fileobj=response) as uncompressed:
-                    file_content = uncompressed.read()
-                    temp_file.write(file_content)
-
-            temp_file_path = temp_file.name
-            onto = get_ontology(temp_file_path).load()
-
```

Comment:
we are no longer going to download each ontology file on the fly. Each owl file will have to be passed individually